types/util.d.ts

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
wdyt about `types/utility-types.d.ts`? Even if redundant it self describes a little better and differentiates more from the four `util.*` files we already have

Reply:
Sure but still exposed as `LH.Util`?